bot/exts/gurkcraft/gurkcraft.py

```diff
@@ -9,6 +9,9 @@
 
 from bot.constants import Channels, Colours, Minecraft
 
+CHAT_HEADER = f"Our gurkan Minecraft server. Join: {Minecraft.server_address} ! \n"
```

Comment:
```suggestion
CHAT_HEADER = f"Our gurkan Minecraft server. Join: {Minecraft.server_address}! \n"
```
I'm not a fan of the space here. Let's throw it into the sun and watch it go up in silver flames.